doc/whatsnew/2.0.rst

```diff
@@ -862,6 +862,14 @@ Bug fixes
   specifications are used in formatted string. In general, these operations
   are not always convertible to old-style formatting used by logging module.
 
+* Added a new switch ``single-line-class-stmt`` to allow single-line declaration
+  of empty class bodies.
```

Comment:
Maybe we should add something as in: ``as seen in this example. Now you can pass this option to ignore this case, similar to single-line-if-stmt``